Revert images.contentDigest back to digest

carolynvs opened the pull request:
# What does this change
This undoes an accidental change from https://github.com/getporter/porter/pull/1308 that changed the manifest
field images.NAME.digest to images.NAME.contentDigest. This is a breaking regression for anyone previously using images. There wasn't any need to change the field name in the manifest, we only needed the bundle.json to use the term contentDigest.

I have changed the manifest field back to digest, and ensured that the user documentation for that field is up to date.

# What issue does it fix
Closes #1355 

# Notes for the reviewer
I've made a follow-up issue to look at tag again. https://github.com/getporter/porter/issues/1357

# Checklist
- [ ] Unit Tests (the existing tests are sufficient)
- [x] Documentation
- [x] Schema (porter.yaml)

vdice commented on docs/content/blog/new-logo.md:
```diff
@@ -0,0 +1,12 @@
+---
```
Is this an older/missed blog post?  Should it be intro'd in a different PR?

carolynvs replied:
oops, that's a draft I had on my computer. Sorry about that!

carolynvs commented:
I have removed the extraneous blog post and fixed the testdata for schema.json. Renaming from contentDigest to digest changed the order of the field declarations.
